### DISCUSSION TITLE
When searching never find anything

### DISCUSSION THREAD

**Discussion by dealelopez** [gsantner/markor#1784] [Q&A] [answered]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**. Not a question or feature request.
- [X] The issue is **not** already reported at [Issues](https://github.com/gsantner/markor/issues?q=) or [discussions](https://github.com/gsantner/markor/discussions?discussions_q=). _(I've searched it)_.
- [X] Markor **is** up to date. See [Releases](https://github.com/gsantner/markor/tags) for the latest version. Updates are available from [F-Droid](https://f-droid.org/en/packages/net.gsantner.markor/) and GitHub.

### Description

When I search for a file the result is always empty. 
Happens in the v2.10.0 (133) and in the previous release.

### Steps to reproduce

1.  Open the search dialog
2. Type a file name that exists
3. Search
4. Nothing found

### Information

Android version: Android 11
Device: Moto G8
App Version: 2.10.0 (133)
 

### Source

F-Droid

### Format / File type

No

### Additional info / Log

```shell
-
```

**Comment by gsantner** [comment, marked as answer]
Have you configured searching in file content, or search only by filename? Check the box depending on the needs, if you search by content you only get files with that content in it.

![screenshot-2022-08-06__14-15-55](https://user-images.githubusercontent.com/6735650/183248536-d74acb92-8fc9-471a-b547-6852c326c9bf.png)
l

**Comment by dealelopez** [comment]
I didn't have the "search in content" option checked.

I have a file called `docker.md`
![Screenshot_20220804-181351](https://user-images.githubusercontent.com/1213588/183443215-0d3552f1-bd38-49bb-8996-1d2dc29cf79c.jpg)
I search for it and nothing is found
![Screenshot_20220804-181405](https://user-images.githubusercontent.com/1213588/183443179-5d89ab4d-b418-485e-a976-e78e85e95d87.png)
![Screenshot_20220804-181351(1)](https://user-images.githubusercontent.com/1213588/183443220-ea189d07-fd41-44f0-bed7-a53805ed51ce.jpg)

Finally I had to clean cache, remove all data and import previous saved config. Now I can't reproduce it.

Thanks for everything
